ui/src/app/routing/workspace-app-routing.tsx

```diff
@@ -205,6 +210,27 @@ export const WorkspaceRoutes = () => {
           leoAppType={LeoApplicationType.JupyterNotebook}
         />
       </AppRoute>
+      <AppRoute
+        exact
+        path={`${path}/gkeApp`}
+        guards={[adminLockedGuard(ns, wsid)]}
+      >
+        <GKEAppRedirectPage
+          key='app'
+          routeData={{
+            pathElementForTitle: 'Gke-App',
```

Comment:
This is invalid, and it's also why the breadcrumb doesn't work.  Let's talk about this.